.azurePipeline/python_init_package_install_steps.yml

```diff
@@ -14,13 +14,18 @@ steps:
       pip install -r requirements.txt
   displayName: 'Install Python requirements'
 
-- script: python setup.py sdist bdist_wheel --universal
+- script: python setup.py sdist bdist_wheel
```

Comment:
We are publishing in Azure some of these build artifacts. Any reason not to build a `universal` one?

Reply:
`universal` means pure python - anonlink contains compiled code